src/core/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,90 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { useTheme } from '../utils/hooks/useTheme';
+import { CommonProps } from '../utils/props';
+import { SvgChevronRight } from '@itwin/itwinui-icons-react';
+import '@itwin/itwinui-css/css/breadcrumbs.css';
+
+export type BreadcrumbsProps = {
+  /**
+   * Index of the currently active breadcrumb.
+   * Defaults to last index in the items array.
+   */
+  currentIndex?: number;
+  /**
+   * Breadcrumb items.
+   */
+  items: JSX.Element[];
+  /**
+   * Specifiy a custom separator element to show between breadcrumb items.
+   */
+  separator?: React.ReactNode;
+} & Omit<CommonProps, 'title'>;
+
+/**
+ * A breadcrumb trail is used as a navigational aid to help users keep track
+ * of their place in the application. It is often placed before a page's main content.
+ *
+ * For accessibility, make sure to provide an aria-label describing the type of navigation.
+ *
+ * @example
+ * <Breadcrumbs
+ *   items={[
+ *     <BreadcrumbItem onClick={() => {}}>Root</BreadcrumbItem>,
+ *     <BreadcrumbItem onClick={() => {}}>Item 1</BreadcrumbItem>,
+ *     <BreadcrumbItem onClick={() => {}}>Item 2</BreadcrumbItem>,
+ *   ]}
+ *   aria-label='breadcrumbs'
+ * />
+ />
+ */
+export const Breadcrumbs = (props: BreadcrumbsProps) => {
+  const {
+    items,
+    currentIndex = items.length - 1,
+    separator,
+    className,
+    ...rest
+  } = props;
+
+  useTheme();
+
+  const Separator = () => (
+    <li className='iui-breadcrumbs-separator' aria-hidden>
```

Comment:
Not sure if `aria-hidden` should go on li element... Usually its applied on svg, will need to read more..

Reply:
The item needs to be hidden from the list, otherwise it will mess with screen readers.